rmats.py

```diff
@@ -417,18 +424,38 @@ def generate_summary(python_executable, out_dir, root_dir):
                         stdout=f_handle)
 
 
+def claim_prep_prefix(task, tmp_dir):
+    if task not in ['prep', 'both']:
+        return None
+
+    file_name_template = os.path.join(tmp_dir, '{}.rmats')
```

Comment:
If https://github.com/Xinglab/rmats-turbo/pull/53 is merged then this would be `'{}_0.rmats'`

Reply:
That change was made in the latest commit which resolves merge conflicts